docs/resources/users_permission.md

```diff
@@ -0,0 +1,85 @@
+# keycloak_users_permissions
+
+Allows you to manage all users Scope Based Permissions.
+
+This is part of a preview keycloak feature. You need to enable this feature to be able to use this resource.
+More information about enabling the preview feature can be found here: https://www.keycloak.org/docs/latest/securing_apps/index.html#_token-exchange
+
+When enabling Users Permissions, Keycloak does several things automatically:
+1. Enable Authorization on build-in realm-management client
+1. Create scopes "view", "manage", "map-roles", "manage-group-membership", "impersonate", "user-impersonated"
+1. Create a resource representing the users permissions
+1. Create all scope based permission for the scopes and openid client resource
+
+If the realm-management Authorization is not enable, you have to ceate a dependency (`depends_on`) with the policy and the openid client.
+
+### Example Usage
+
+```hcl
+resource "keycloak_realm" "realm" {
+	realm  = "realm"
+}
+
+data "keycloak_openid_client" "realm_management" {
+	realm_id  = keycloak_realm.realm.id
+	client_id = "realm-management"  
+}
+
+resource keycloak_openid_client_permissions "realm-management_permission" {
+	realm_id   = keycloak_realm.realm.id
+	client_id  = data.keycloak_openid_client.realm_management.id
+	enabled = true
+}
+
+resource keycloak_user test {
+	realm_id = "${keycloak_realm.realm.id}"
```

Comment:
Can you update this example to remove the deprecated interpolation syntax?  In this case, we should change `"${keycloak_realm.realm.id}"` to `keycloak_realm.realm.id`